course_info/tests/jasmine/controllers/DetailsControllerSpec.js

```diff
@@ -13,6 +13,14 @@ describe('Unit testing DetailsController', function () {
         '=api%2Fcourse%2Fv2%2Fcourse_instances%2F' + courseInstanceId +
         '%2Fpeople%2F&-source=xreg_map';
 
+    var deleteSiteUrl =
```

Comment:
In the future since we're bringing the djangoUrl module into our test framework we can probably just use its `reverse()` function to make the code more concise (see #142). No need right now.